satellite/repair/repairer/peer.go

```diff
@@ -0,0 +1,169 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package repairer
```

Comment:
For peers it probably makes more sense to keep them at a higher directory, since they are top-level concepts. For now I would keep the peers related to satellite together rather than splitting them out to several packages.